play/symphony-conductor/index.html

```diff
@@ -1,130 +1,243 @@
+                <p>
+                    This allows you to create a link that contains all of your code. You can then send/share this link
+                    wherever, and if someone clicks on the link, it will open up this website with your code already
+                    loaded into the editor.
+                </p>
+
+                <p>
+                    <button class="btn" onclick="shareCode()">Share</button>
+                </p>
+
+                <h3 id="label10">Video Tutorial</h3> <h3 id="label11">Examples</h3>
+                <!-- <p id="exampleLine">⬇️ Checkout Some Cool Melodies Examples !!! ⬇️ </p> -->
+
+
+                <textarea name="Text1" cols="16" rows="15" id="myInput0">
+/* Example */
+
+play("DJ-Scrubbing");
+await rest(1);
+play("Acoustic-Guitar");
+await rest(2);
+play("snare");
+await rest(2);
+play("snare");
+await rest(2);
+play("snare");
+await rest(.2);
+play("Hand-Clap");
+play("snare");
+await rest(.4);
+play("Hand-Clap");
+                </textarea>
+
+                <textarea name="Text1" cols="16" rows="15" id="myInput1">
+/* Example */
+
+play("trumpet");
+await rest(1);
+
+play("Xylophone");
+play("Hand-Clap");
+await rest(.5);
+play("Xylophone");
+play("Hand-Clap");
+
+await rest(2);
+
+play("Xylophone");
+play("Hand-Clap");
+await rest(.5);
+play("Xylophone");
+play("Hand-Clap");
+
+                    </textarea>
+                <textarea name="Text2" cols="16" rows="15" id="myInput2">
+/* Example */
+
+play("snare");
+await rest(.5);
+play("snare");
+await rest(.2);
+play("Hand-Clap");
+await rest(.6);
+
+play("snare");
+await rest(.5);
+play("snare");
+await rest(.2);
+play("Hand-Clap");
+await rest(.6);
+
+play("snare");
+await rest(.35);
+play("snare");
+await rest(.2);
+play("Hand-Clap");
+play("Electric-Guitar");
+
+await rest(.6);
+
+play("snare");
+await rest(.35);
+play("snare");
+await rest(.2);
+play("Hand-Clap");
+await rest(.6);
                         
-                    <h3>Share Code With URL</h3>
-                    <p>
-                        This allows you to create a link that contains all of your code. You can then send/share this link wherever, and if someone clicks on the link, it will open up this website with your code already loaded into the editor.
-                    </p>
-                    
-                    <p>
-                        <button class="btn" onclick="shareCode()">Share</button>
-                    </p>
+                    </textarea>
+
+                <button onclick="myFunction(`myInput0`)" id="exampleButton0">Copy (Beginner Level)</button>
+
+                <button onclick="myFunction(`myInput`)" id="exampleButton1">Copy (Intermediate Level)</button>
+
+                <button onclick="myFunction(`myInput2`)" id="exampleButton2">Copy (Complex Level)</button>
+
+                <video width="329" height="190" controls id="tutorialVideo">
+                    <source src="TutorialVideo.mp4" type="video/mp4">
```

Comment:
I uploaded this file to YouTube in order to not have to commit such a large video file. Could you change this to the YouTube URL: https://youtu.be/0kg-ZQXIpFY

Reply:
I switched the video src from the mp4 file to the youtube link